content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -661,6 +661,67 @@ of single-resource API requests, then aggregates the responses if needed.
 By contrast, the Kubernetes API verbs **list** and **watch** allow getting multiple
 resources, and **deletecollection** allows deleting multiple resources.
 
+## Field Validation
+
+  {{< feature-state for_k8s_version="v1.24" state="beta" >}}
+
+In certain situations, the API server will drop fields it does not recognize
+from an object it receives.
+
+The two situations where this will occur are:
+
+1. The field is unrecognized because it is not in the resource's OpenAPI schema. (One
+   exception to this is for CRDs that explicitly choose not to prune unknown
+   fields via `x-kubernetes-preserve-unknown-fields`).
+2. The field is duplicated in the object.
+
+When you use HTTP verbs that can submit data (`POST`, `PUT`, and `PATCH`), field
+validation gives you the option to choose how you would like to be notified of
+these fields that are being dropped by the API server. Possible levels of
+validation are `Ignore`, `Warn`, and `Strict`.
+
+{{< note >}}
+Any fields that are erroneous for other reasons (i.e. a
+field that is supposed to be an integer, but is submitted as a string), will always
+result in the request failing with a bad request error.
+{{< /note >}}
+
+### Setting the field validation level
+
+Field validation is set by the `fieldValidation` query parameter. This parameter
+is a string, working as an enum and the only accepted values are:
```

Comment:
How about:
```suggestion
Field validation is set by the optional `fieldValidation` query parameter. The three
values that you can provide for this parameter are:
```

I don't think we need to tell the reader that `Ignore`, `Warn` and `Strict` are strings; either the reader is amply familiar with strings in this context, enough to work it out for themselves, or they have a lot more background reading to do before it's useful to explain that detail.

For a client, these are opaque values, and anyway, substrings of URLs are always strings.